sig-usability/README.md

```diff
@@ -0,0 +1,52 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Usability Special Interest Group
+
+The scope of SIG usability is the core end-user usability of the Kubernetes project. Thiis covers topics like user experience and accessibility. The goals of the SIG are to ensure that the Kubernetes project is maximally usable by users of a variety of different backgrounds and abilities. Examples of efforts include internationalization and accessibility of documentation.
+
+The [charter](charter.md) defines the scope and governance of the Usability Special Interest Group.
+
+## Meetings
+* Regular SIG Meeting: [TBDs at TBD PT (Pacific Time)](tbd) (every third week). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=TBD&tz=PT%20%28Pacific%20Time%29).
+  * [Meeting notes and Agenda](tbd).
+  * [Meeting recordings](tbd).
+
+## Leadership
+
+### Chairs
+The Chairs of the SIG run operations and processes governing the SIG.
```

Comment:
Is everyone part of the k8s org, if not, please propose them?

Reply:
@gaby-moreno @vllry @hpandeycodeit @tashimi @Rajakavitha1 

Can you confirm that you are all already members of the Kubernetes organization? If you are not, and you think you qualify for nomination, please let me know and we can start to push that forward. Otherwise, I think we'll just have you be informal chairs for now, and then you can use your participation in the SIG to become a full fledged member of the organization.

Reply:
Yes, I am a member of the Kubernetes organization.

Reply:
I'm a member, as is @tashimi and @hpandeycodeit. It looks like @gaby-moreno is not.

Reply:
Hi @brendanburns, I am not a member yet

Reply:
Yeah, I'm a member. :)

Reply:
@gaby-moreno

For now, I've removed you from the list of chairs, not because we don't want you to be a chair, but just because it will be easier to form this up and then add you later.

Please review the requirements for membership here:
https://github.com/kubernetes/community/blob/master/community-membership.md#requirements

If you think that you have met the requirements, please find two sponsors (I'm happy to be one of them) and we'll get you into the org.

Otherwise, active participation in this SIG will certainly qualify you for membership in the org.

Please let me know if there are questions! Thanks!

Reply:
Hi @brendanburns, that's fine. Thank you for the resource and for offering to sponsor!